### PR TITLE
[TEP-0116] implemented

### PR DESCRIPTION
**Pull request by vsinghai**
TEP-0116 has been implemented.
The status has changed from `implementable` to 
`implemented`.

### CODE REVIEW

**Review by dibyom — APPROVED** [review]

File: teps/0116-referencing-finally-task-results-in-pipeline-results.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-status: implementable
+status: implemented
 title: Referencing Finally Task Results in Pipeline Results
 creation-date: '2022-06-24'
 last-updated: '2022-07-15'
```
Comment (jerop) [inline teps/0116-referencing-finally-task-results-in-pipeline-results.md:5]: please update this too

**Review by jerop — APPROVED** [review]
congratulations @vsinghai 🎉

File: teps/0116-referencing-finally-task-results-in-pipeline-results.md (hunk 1 above)
Comment (vsinghai) [reply teps/0116-referencing-finally-task-results-in-pipeline-results.md]: done

**Pull request merged**